<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditColorDifferenceLibraryDialog</class>
 <widget class="QDialog" name="EditColorDifferenceLibraryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ButtonLibSaveNew">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>500</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save New</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Qtres/SaveNew.png</normaloff>:/Qtres/SaveNew.png</iconset>
   </property>
  </widget>
  <widget class="QFrame" name="frameLibFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>閉じる</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Qtres/Close.png</normaloff>:/Qtres/Close.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLibDelete">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Qtres/Delete.png</normaloff>:/Qtres/Delete.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLibSave">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>500</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetLibList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="EditLibName">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Library Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLibNew">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>500</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear new</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Qtres/ClearNew.png</normaloff>:/Qtres/ClearNew.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibID">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="toolButtonItem">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>通常色調領域</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonReference">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>参照領域</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>441</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>生成</string>
    </attribute>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxGenDeltaE">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_61">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>最大サイズ（長さ）</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_60">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>領域生成のΔE範囲</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMaxLength">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_62">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>分解矩形のの長さ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxGenBlockResolution">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_63">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>最小面積</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxGenMinArea">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_64">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>最大面積</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxGenMaxArea">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxGenMaxVariable">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_65">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>同一化できるばらつき最大値</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxGenMaxBrightDifference">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>200</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_66">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>同一化できる輝度差最大値</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_67">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>収縮ドット数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxGenSurroundMergin">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>230</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>検査</string>
    </attribute>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>411</width>
       <height>111</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>111</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;検&lt;/p&gt;&lt;p&gt;査&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_49">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>241</width>
        <height>24</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>採用率(%)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxAdoptedRate">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxTHDeltaE">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_52">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>241</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>ΔEしきい値</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_56">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>判定方法</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxJudgeMethod">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>231</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>OK/NGを使う</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DeltaEで判定</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>流動サンプルでHSV判定</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>411</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;H&lt;/p&gt;&lt;p&gt;S&lt;/p&gt;&lt;p&gt;V&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_50">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>241</width>
        <height>24</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Hの許容差異</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxdH">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxdSL">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_51">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>241</width>
        <height>24</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Sの中心側の許容差異</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxdSH">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_53">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>241</width>
        <height>24</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Sの外側の許容差異</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxdVL">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_54">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>241</width>
        <height>24</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Vの暗側の許容差異</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxdVH">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_55">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>241</width>
        <height>24</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Vの明側の許容差異</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_57">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>位置合わせで移動する</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAdaptAlignment">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>300</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
